tests/scripts/run_performance.sh

```diff
@@ -61,6 +61,8 @@ run_perf_models_llm_javelin() {
 
     if [ "$tt_arch" == "wormhole_b0" ]; then
         env pytest models/demos/mistral7b/tests -m $test_marker
+
+        env pytest models/demos/mamba/tests -m $test_marker
```

Comment:
Can you please put this command above mistral7b and post a model perf e2e regressions pipeline link where this is passing?